### DISCUSSION TITLE
release boolean editing directly on the grid

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#6159] [Help] [answered]
Esteemed,
How to edit the checkbox of the boolean field directly in the grid and then save it, as with the other fields.

![image](https://user-images.githubusercontent.com/7516443/147271901-577c459c-2faf-4242-8f61-05490b2d13b0.png)

**Comment by wezmag** [comment, marked as answer]
Check out this pull request.
https://github.com/serenity-is/common-features/pull/8

**Reply by juliermefelix** [reply]
Thanks
